authz.go

```diff
@@ -468,6 +495,11 @@ func MePermissions(userID platform.ID) []Permission {
 func MemberPermissions(orgID platform.ID) []Permission {
 	ps := []Permission{}
 	for _, r := range AllResourceTypes {
+		// For now, we are only allowing instance permissions when logged in through session auth
```

Comment:
Please remember to remove these 'For now' comments later.

Reply:
These comments (as well as the logic) would go away if/when we add token support, unless I'm misunderstanding something?